README.md

```diff
@@ -57,7 +62,18 @@ When relevant, we encourage you to write tests. You can run the tests using the
 In addition to the code tests there are also visual regression tests, located in the `./cypress/integration` directory. You can run these tests locally by installing [cypress](https://www.cypress.io/) using `npm i cypress@3.0.3`, after which the command `npm run cypress` will run these tests locally. However, note that these tests are currently intended for screenshot comparisons across branches, and so will not yield any meaningful results when run for a single branch.
 
 
+## How to Setup your Dev Environment with Docker
+
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) (macOS and Windows). For Linux users: install [Docker CE](https://docs.docker.com/install/#supported-platforms) and [Docker Compose](https://docs.docker.com/compose/install/),
+- [Check your install](https://docs.docker.com/get-started/#test-docker-version) by running `docker run hello-world`,
+- If relevant: delete your node_modules directory (`rm -rf node_modules`). It's not necessary, but it speeds up the install.
+- Run `invoke docker-setup` ([install invoke](http://www.pyinvoke.org/installing.html) if you don't have it yet). If you're running on Windows, you need to run `docker-compose --rm pipenv run python network-api/manage.py createsuperuser` when the setup is finished.
```

Comment:
can't we alias that extra windows step with invoke? `invoke docker-setup-windows`?

Reply:
Sadly no, for the same reason it was not possible to do it before.